Content.Shared/CombatMode/CombatModeComponent.cs

```diff
@@ -37,9 +37,8 @@ public sealed partial class CombatModeComponent : Component
         private TargetingZone _activeZone;
 
         [DataField("combatToggleActionId", customTypeSerializer: typeof(PrototypeIdSerializer<InstantActionPrototype>))]
-        public string CombatToggleActionId = "CombatModeToggle";
+        public string? CombatToggleActionId = "CombatModeToggle";
 
-        [DataField("combatToggleAction")]
         public InstantAction? CombatToggleAction;
```

Comment:
this was causing tests to fail because it was resaving the `CombatToggleAction` since ballistic turrets arent marked as nosave... just removed the datafield and made the id nullable instead